packages/syft/src/syft/user_settings.py

```diff
@@ -0,0 +1,41 @@
+# future
+from __future__ import annotations
+
+# stdlib
+import os
+
+# third party
+from dotenv import load_dotenv
+
+# relative
+from .core.common.decorators import singleton
+
+# Get root path of project
```

Comment:
Let's remove the .env loading stuff for now this looks like something that might cause some issues. I think we want something like ~/.syft/settings.json but lets look into it as a seperate task. For now, lets put SYFT_TUTORIAL_MODE=False in all the docker container environment variables and later when we notice this in notebooks we can add something like sy.settings.helper = False or something.

Reply:
@bitsofsteve

Reply:
Okay! @madhavajay